benchmarks/run_offline.py

```diff
@@ -62,13 +65,18 @@
 _QUANTIZE_WEIGHTS = flags.DEFINE_bool(
     "quantize_weights", False, "weight quantization"
 )
+_QUANTIZE_NUM_BITS_WEIGHTS = flags.DEFINE_integer(
```

Comment:
combine these into `quantize_type` which can take values of int8, int4, int4_block

Reply:
Agree, done